tokens/src/lib.rs

```diff
@@ -43,6 +43,7 @@ use frame_support::{
 	ensure, log,
 	pallet_prelude::*,
 	traits::{
+		tokens::{fungible as PalletFungible, fungibles as PalletFungibles, DepositConsequence, WithdrawConsequence},
```

Comment:
Is `fungible as PalletFungible, fungibles as PalletFungibles` renaming really required? They should stay snake cases.